src/mlpack/core/optimizers/sgd/update_policies/vanilla_update.hpp

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file vanilla_update.hpp
+ * @author Arun Reddy
+ *
+ * Empty update for SGD
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_SGD_EMPTY_UPDATE_HPP
+#define MLPACK_CORE_OPTIMIZERS_SGD_EMPTY_UPDATE_HPP
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * Vanilla update policy for SGD.
+ *
+ */
+class VanillaUpdate {
+ public:
+
+  /**
+   * Function for runtime initialization.
+   *
+   * @param n_rows
```

Comment:
We should add a description for each parameter. That also applies for the Update function.